tests/test_geom.py

```diff
@@ -44,6 +45,13 @@ def test_invalid_mesh_area_names(tmp_path):
         assert ghdf.mesh_area_names() == []
 
 
+def test_missing_mesh_in_mesh_area():
+    ras_hdf = RasGeomHdf(BLW_WEST_FORK)
+    expected_error_message = "Mesh blw-west-fork not found."
+    with pytest.raises(Exception, match=expected_error_message):
+        ras_hdf.mesh_areas()
```

Comment:
See comments on `blw-west-fork.g04.hdf`. Check out the `test_projection` test for an example which creates a dummy HDF file.